Pear/Controller/CommunityViewController.swift

```diff
@@ -143,17 +152,33 @@ extension CommunityViewController: UITableViewDataSource {
 
 // MARK: - SearchBarDelegate
 extension CommunityViewController: UISearchBarDelegate {
-
-    private func searchUsers(query: String) {
-        // TODO: Fill in with network call
-    }
-
-    func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
-        if searchText.isEmpty {
-            getUsers()
-        } else {
+    
+    func getUsers() {
+        if let searchText = searchBar.text {
             searchUsers(query: searchText)
+        } else {
+            getAllUsers()
         }
     }
 
+    func searchUsers(query: String) {
+        NetworkManager.getSearchedUsers(searchText: query, completion: { [weak self] result in
```

Comment:
Because this is a trailing closure, you should be able to omit the argument name.

```suggestion
        NetworkManager.getSearchedUsers(searchText: query) { [weak self] result in
```